cloudfront/edge-lambda/dlcs_path_rewrite.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+// lambda@edge Origin Request trigger to remove the first path element
+
+exports.request = (event, context, callback) => {
+    const request = event.Records[0].cf.request;           // extract the request object
+    request.uri = request.uri.replace("/image/","/");      // remove leading /image/ from iiif URL
+    request.uri = request.uri.replace(".jpg/","/");        // remove any non path terminating .jpg from iiif URL
+    return callback(null, request);                        // return control to CloudFront
```

Comment:
Might be worth adding some examples of the URLs you're editing here?

We know them now, but it'll be less obvious once Loris is decommissioned and the only people hitting this code are those with saved URLs.